modules/Bio/EnsEMBL/Variation/Pipeline/SpliceAI/SpliceAI_conf.pm

```diff
@@ -0,0 +1,195 @@
+            -driver => 'mysql',
+        },
+    };
+}
+
+sub resource_classes {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::resource_classes},
+        '4Gb_8c_job'  => {'LSF' => '-n 8 -q production-rh74 -R"select[mem>4000]  rusage[mem=4000]" -M4000' },
+        '4Gb_job'     => {'LSF' => '-q production-rh74 -R"select[mem>4000] rusage[mem=4000]" -M4000'},
+    };
+}
+
+sub pipeline_analyses {
+  my ($self) = @_;
+  my @analyses;
+  push @analyses, (
+      # pre run checks, directories exist etc
+
+      {   -logic_name => 'init_files',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+          -input_ids  => [{}],
+          -parameters => {
+            'input_directory' => $self->o('input_directory'),
+            'inputcmd'        => 'find #input_directory# -type f -name "all_snps_ensembl_38_*.vcf" -printf "%f\n"',
+          },
+          -flow_into  => {
+            '2->A' => {'split_files' => {'vcf_file' => '#_0#'}},
+            'A->1' => ['get_chr_dir'],
+          },
+      },
+      { -logic_name => 'split_files',
+        -module => 'Bio::EnsEMBL::Variation::Pipeline::SpliceAI::SplitFiles',
+        -input_ids  => [],
+        -parameters => {
+          'main_dir'          => $self->o('main_dir'),
+          'input_directory'   => $self->o('input_directory'),
+          'tmp_split_vcf_dir' => $self->o('tmp_split_vcf_dir'),
+          'split_vcf_dir'     => $self->o('tmp_split_vcf_input_dir'),
+          'output_dir'        => $self->o('tmp_output_dir'),
+          'step_size'         => $self->o('step_size'),
+          'check_transcripts' => $self->o('check_transcripts'),
+          # 'transcripts_file'  => $self->o('transcripts_file'),
+          'registry'          => $self->o('registry'),
+        },
+      },
+      { -logic_name => 'get_chr_dir',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+        -input_ids  => [],
+        -parameters => {
+          'input_dir' => $self->o('tmp_split_vcf_input_dir'),
+          'inputcmd'  => 'ls #input_dir#',
+        },
+        -flow_into => { 
+          '2->A' => {'init_spliceai' => {'input_chr_dir' => '#_0#'}},
+          'A->1' => ['init_merge_files'],
+        },
+      },
+      {   -logic_name => 'init_spliceai',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+          -input_ids  => [],
+          -parameters => {
+            'input_dir' => $self->o('main_dir') . '/tmp_split_vcf_input/' . '#input_chr_dir#',
+            'inputcmd'  => 'find #input_dir# -type f -name "all_snps_ensembl_38_*.vcf" -printf "%f\n"',
+          },
+          -flow_into  => {
+            2 => {'run_spliceai' => {'input_file' => '#_0#'}},
+          }
+      },
+      { -logic_name => 'run_spliceai',
+        -module => 'Bio::EnsEMBL::Variation::Pipeline::SpliceAI::RunSpliceAI',
+        # -hive_capacity => $self->o('pipeline_wide_analysis_capacity'),
```

Comment:
Why is the hive_capacity not used? Can it be deleted?

Reply:
I probably forgot I had added it as a comment. It should be used.